app/experimenter/experiments/tests/api/v3/test_serializers.py

```diff
@@ -262,7 +220,7 @@ def test_serializer_bad_audience_value(self):
             "objectives": "gotta go fast",
             "audience": "WRONG AUDIENCE CHOICE",
             "features": ["picture_in_picture", "pinned_tabs"],
-            "firefox_min_version": FIREFOX_VERSION,
+            "firefox_min_version": "80.0",
```

Comment:
Why not use the constant or `experiment.firefox_min_version` here?

Reply:
Ya good idea 👍

Reply:
Oh no that's data going in, there is no `experiment` in that context, it's all just literals.

Reply:
I had that constant up at the top of the file for some reason at some point but it's not generally a pattern we do and I didn't want to encourage it since test constants should be scoped to the test method or test class, so I wanted to get rid of it.  What do you think?  Mind you we're getting rid of these tests when we add graphql soon.